### ISSUE TITLE
Random properties for each of the starting regions.

### ISSUE THREAD

**Issue by GoogleCodeExporter** [sunfall/endgame-singularity#18]
```
The starting regions should have some random properties to make each
location distinct.  Phil suggested creating a list of properties and
randomly distributing them, to ensure balance.

Possible positive properties:
CPU bonus
Bonus vs detection to one/all groups
Lowered construction/maintenance cost (CPU/Cash)
Increased build speed (Labor)

Negative properties can simply be the inverse of the positive ones.
```

Original issue reported on code.google.com by `funnyman3595` on 29 Apr 2008 at 6:04

**Comment by GoogleCodeExporter** [comment]
```
Note that each location should have both positives and negatives, so it's never 
just
"pick the good places and ignore the bad ones."
```

Original comment by `phil.bor...@gmail.com` on 29 Apr 2008 at 6:07

**Comment by GoogleCodeExporter** [comment]
```
It might work to have one location with only a positive.  The discovery% 
increase
from having bases discovered there would deter camping it.
```

Original comment by `funnyman3595` on 29 Apr 2008 at 6:35

**Comment by GoogleCodeExporter** [comment]
Original comment by `funnyman3595` on 29 Apr 2008 at 6:48
* Added labels: **Component-Logic**

**Comment by GoogleCodeExporter** [comment]
Original comment by `funnyman3595` on 29 Apr 2008 at 7:11
* Added labels: **Milestone-0.28**

**Comment by GoogleCodeExporter** [comment]
```
Implemented in git (b9fba0b) and SVN (r618).
```

Original comment by `funnyman3595` on 2 May 2008 at 2:19
* Changed state: **Fixed**
